Mathlib/CategoryTheory/MorphismProperty/Presheaf.lean

```diff
@@ -0,0 +1,202 @@
+/-
+Copyright (c) 2024 Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne, Joël Riou, Ravi Vakil
+-/
+
+import Mathlib.CategoryTheory.Limits.Shapes.Pullback.CommSq
+
+/-!
+
+# Representable morphisms of presheaves
+
+In this file we define and develop basic results on the representability of morphisms of presheaves.
+
+## Main definitions
+
+* `Presheaf.representable` is a `MorphismProperty` expressing the fact that a morphism of presheaves
+  is representable.
+
+## API
+
+Given `hf : Presheaf.representable f`, with `f : F ⟶ G` and `g : yoneda.obj X ⟶ G`, we provide:
+* `hf.pullback g` which is the object in `C` such that `yoneda.obj (hf.pullback g)` forms a
+  pullback square of `f` and `g`.
+* `hf.snd g` is the morphism `hf.pullback g ⟶ X`
+* `hf.fst g` is the morphism `yoneda.obj (hf.pullback g) ⟶ F`
+*  Whenever `f` is of type `yoneda.obj Y ⟶ G`, we also have `hf.fst' g` which is the preimage
+   under `yoneda` of `hf.fst g`.
+* `hom_ext`, `hom_ext'`, `lift`, `lift'` are variants of the universal property of
+  `yoneda.obj (hf.pullback g)`, where as much as possible have been formulated internally to `C`.
+
+* `symmetry` and `symmetryIso` are variants of the fact that pullbacks are symmetric for
+  representable morphisms, formulated internally to `C`.
+
```

Comment:
Is there a reason for this blank line between bullet points? If not, could you remove it?

Reply:
I think in my head these blank lines separated different categories somehow, but now that I look back on it I think that no blank line is better